packages/util-utf8/jest.config.js

```diff
@@ -0,0 +1,5 @@
+const base = require("../../jest.config.base.js");
+
+module.exports = {
+  ...base,
```

Comment:
Override required to ensure tests in `*.browser.spec.ts` are run.
```suggestion
  ...base,
  testMatch: ["**/*.spec.ts"],
```

Reply:
fixed